android/views/src/main/kotlin/com/boswelja/ephemeris/views/CalendarPagerAdapter.kt

```diff
@@ -22,8 +22,7 @@ internal class CalendarPagerAdapter(
     }
 
     override fun onBindViewHolder(holder: CalendarPageViewHolder, position: Int) {
-        val page = (position - (Int.MAX_VALUE / 2)).toLong()
-        val pageState = pagingSource.loadPage(page)
+        val pageState = pagingSource.loadPage(position.toLong())
```

Comment:
This has introduced a regression where the user cannot scroll backwards from the starting index

Reply:
Ohh that's what was causing, this might need a calculation like `pagesFromEpoch` or something along those lines to figure out an acceptable min value?

Reply:
I'm wondering if the original bug was simply that the RecyclerView did not start from Int.MAX_VALUE/2